src/main/java/frc/utils/applicationsutils/OutputFile.java

```diff
@@ -0,0 +1,97 @@
+package frc.utils.applicationsutils;
+
+import frc.robot.constants.DirectoryPathsConstants;
+
+import java.awt.*;
+import java.io.File;
+import java.io.FileWriter;
+import java.nio.file.Files;
+import java.nio.file.StandardOpenOption;
+
+public class OutputFile {
+
+    private static final File SELF_OUTPUT_FILE = getSelfOutputFile();
```

Comment:
Can you explain to me why its called Self output file?

Reply:
because it is it's own output file?
i also cant name it OUTPUT_FILE_OUTPUT_FILE

Reply:
this file is the output file for any errors happening in this class.
that is why it is called an output file

Reply:
I really prefer it to be Files Errors output file, when the name point to what will be in the file instead of who the file belong to